### DISCUSSION TITLE
Continuous Delivery of NuGet Packages in Azure Artifacts - v5 or v6?

### DISCUSSION THREAD

**Discussion by thepaulmacca** [GitTools/GitVersion#4351] [General]
Hi, it's been a good while since I've been tasked with this - but when working with NuGet packages previously I used [this article](https://www.microsoft.com/en-gb/industry/blog/technetuk/2019/06/18/perfecting-continuous-delivery-of-nuget-packages-for-azure-artifacts/) as a good reference point

Is it still relatively safe to use GitVersion v5 for the forseeable, or is there a v6 equivalent that someone could provide me by example please? I'm struggling to get my head around it at the moment, given all the breaking changes in v6 - so if there was an example that I could use to test things out, it'd be much appreciated!

Thanks

**Comment by arturcic** [comment]
> Hi, it's been a good while since I've been tasked with this - but when working with NuGet packages previously I used [this article](https://www.microsoft.com/en-gb/industry/blog/technetuk/2019/06/18/perfecting-continuous-delivery-of-nuget-packages-for-azure-artifacts/) as a good reference point
> 
> Is it still relatively safe to use GitVersion v5 for the forseeable, or is there a v6 equivalent that someone could provide me by example please? I'm struggling to get my head around it at the moment, given all the breaking changes in v6 - so if there was an example that I could use to test things out, it'd be much appreciated!
> 
> Thanks

v5 is no longer supported, so I would recommend starting using v6 as this is the version we're working on

**Reply by thepaulmacca** [reply]
Ok thanks, do you have a very rough v6 example of the v5 equivalent that you could share for NuGet packages?

```
mode: Mainline 
next-version: 1.0.0 
branches:
 feature:
   tag: alpha
 master:
 tag: '' 
ignore:
 sha: []
```

**Reply by HHobeck** [reply]
> Ok thanks, do you have a very rough v6 example of the v5 equivalent that you could share for NuGet packages?

```yaml
workflow: TrunkBased/preview1
```

https://gitversion.net/docs/reference/configuration#snippet-/docs/workflows/TrunkBased/preview1.yml
